providers/slack/resources/conversations.go

```diff
@@ -7,14 +7,19 @@ import (
 	"errors"
 	"time"
 
+	"github.com/rs/zerolog/log"
 	"github.com/slack-go/slack"
 	"go.mondoo.com/cnquery/v9/llx"
 	"go.mondoo.com/cnquery/v9/providers-sdk/v1/plugin"
 	"go.mondoo.com/cnquery/v9/providers-sdk/v1/util/convert"
 	"go.mondoo.com/cnquery/v9/providers/slack/connection"
 )
 
-func (s *mqlSlack) conversations() ([]interface{}, error) {
+func (o *mqlSlackConversations) id() (string, error) {
+	return "slack.conversations", nil
+}
+
+func (s *mqlSlackConversations) listChannels(types []string) ([]interface{}, error) {
```

Comment:
nit: if we did `listChannels(types ...string)` here we could call it with `listChannels("private_channel")` etc below; but really just a nit, no real impact...

Reply:
I like it and adjust the code

Reply:
fixed